example/cumulus-tf/kinesis_trigger_test_workflow.asl.json

```diff
@@ -125,7 +124,92 @@
         }
       },
       "Type": "Task",
-      "Resource": "${cnm_response_task_arn}",
+      "Resource": "${fake_processing_task_arn}",
+      "Next": "FilesToGranulesStep",
+      "Catch": [
+        {
+          "ErrorEquals": [
+            "States.ALL"
+          ],
+          "Next": "CnmResponseFail",
+          "ResultPath": "$.exception"
+        }
+      ],
+      "Retry": [
+        {
+          "BackoffRate": 2,
+          "ErrorEquals": [
+            "Lambda.ServiceException",
+            "Lambda.AWSLambdaException",
+            "Lambda.SdkClientException"
+          ],
+          "IntervalSeconds": 2,
+          "MaxAttempts": 6
+        }
+      ]
+    },
+    "FilesToGranulesStep": {
```

Comment:
These tasks are added so that `CnmResponse` will work.  `CnmResponse` expects the 1) input has granules 
2) for sucessful response, it also expects granule to have cmr information and granule file has size.

https://github.com/podaac/cumulus-cnm-response-task/blob/develop/src/main/java/gov/nasa/cumulus/CNMResponse.java#L200
https://github.com/podaac/cumulus-cnm-response-task/blob/develop/src/main/java/gov/nasa/cumulus/CNMResponse.java#L136-L137
https://github.com/podaac/cumulus-cnm-response-task/blob/develop/src/main/java/gov/nasa/cumulus/CNMResponse.java#L149-L150